core/trino-spi/src/main/java/io/trino/spi/predicate/EquatableValueSet.java

```diff
@@ -303,9 +304,27 @@ public String toString()
     @Override
     public String toString(ConnectorSession session)
     {
-        return (inclusive ? "[ " : "EXCLUDES[ ") + entries.stream()
-                .map(entry -> type.getObjectValue(session, entry.getBlock(), 0).toString())
-                .collect(Collectors.joining(", ")) + " ]";
+        return toString(session, 10);
+    }
+
+    @Override
+    public String toString(ConnectorSession session, int limit)
+    {
+        return new StringJoiner(", ", EquatableValueSet.class.getSimpleName() + (inclusive ? "[" : " EXCLUDES["), "]")
```

Comment:
"EquatableValueSet" shows up on output

- the change certainly doesn't belong to `Add limit to Domain#toString SPI` commit
- it's odd direction for this PR (you seem to be simplifying output in general), but i am fine with the change

Reply:
I added it for consistency with SortedRangeSet, alternatively I would remove class name in both places.
I'll pull it out into a separate commit